test/state/state.cpp

```diff
@@ -311,7 +311,7 @@ std::variant<int64_t, std::error_code> validate_transaction(const StateView& sta
         if (tx.blob_hashes.empty())
             return make_error_code(EMPTY_BLOB_HASHES_LIST);
 
-        if (tx.max_blob_gas_price < block.blob_base_fee)
+        if (tx.max_blob_gas_price < block.blob_base_fee.value())
```

Comment:
```suggestion
        if (tx.max_blob_gas_price < *block.blob_base_fee)
```

Reply:
I would also add assertion checking that `block.block_base_fee.has_value()`